src/java/com/unifina/feed/AbstractFeedProxy.java

```diff
@@ -164,7 +164,7 @@ private void processAndQueue(long counter, MessageClass msg, boolean checkAge) {
 
 			// TODO: remove debug
 			if (firstRealQueue == null && checkAge) {
-				log.info("First real time message: " + counter + ".");
+				log.info("First real time message: " + counter + ". Event count: " + events.length);
```

Comment:
Potential NPE?

Reply:
Apparently yes, since there is a null check later on in the same code section. (Though pretty bad practice to allow null to be returned by the `process` method.)

Reply:
@jtakalai

Reply:
I simply removed whole log call based on TODO comment above.